Header header
std_msgs/UInt64 keyFrameId
sensor_msgs/Image keyFrameImage